templates/init.erb

```diff
@@ -107,13 +113,13 @@ stop() {
 }
 
 status() {
-  if [ ! -f "$PID_FILE" ]; then
+  if ! [ -f "$PID_FILE" ]; then
```

Comment:
FWIW I don't think this matters.   `[` is equiv to the builtin `test`  and from the [posix description of test](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html):

> !  expression
True if expression is false. False if expression is true.

```
$ [ -f /doesnotexist ] && echo true || echo false
false
$ [ ! -f /doesnotexist ] && echo true || echo false 
true
```

I'm curious if you just did this for style purposes, habit, or some other need that I'm missing?